elasticsearch-extensions/lib/elasticsearch/extensions/test/cluster.rb

```diff
@@ -170,10 +170,22 @@ def stop(arguments={})
           unless pids.empty?
             print "\nStopping Elasticsearch nodes... ".ansi(:faint)
             pids.each_with_index do |pid, i|
-              begin
-                print "stopped PID #{pid}. ".ansi(:green) if Process.kill 'INT', pid
-              rescue Exception => e
-                print "[#{e.class}] PID #{pid} not found. ".ansi(:red)
+              ['INT','KILL'].each do |signal|
+                begin
+                  print "stopped PID #{pid} with #{signal} signal. ".ansi(:green) if Process.kill signal, pid
```

Comment:
huh. 

I don't disagree with you, just wonder how did it ever work? The logic in that line has been there since before I ever looked at this code.

I will rework it.